Calculator/Calculator/CalculatorTest/FormulaTest/FormulaTest.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  FormulaTest.swift
+//  FormulaTest
+//
+//  Created by Lee minyeol on 2023/10/11.
+//
+
+import XCTest
+@testable import Calculator
+
+final class FormulaTest: XCTestCase {
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+    }
```

Comment:
sut를 설정하는 코드가 없네요!
Formula의 메서드를 테스트할 것이기 때문에 sut는 Formula가 되어야하겠죠?!
저는 테스트의 용이성을 위해 아래와 같이 프로퍼티를 주입시켜 주는 편입니다.
이 파일 전반적으로 수정이 필요해보입니다~ 💪

테스트케이스는 잘 작성하신 것 같습니다 👍👍👍
다만, throw하는 메서드의 경우는 예외 상황에서 제대로 오류를 던지는지 테스트할 필요도 있습니다. 🤔

```suggestion
final class FormulaTest: XCTestCase {
    var operands: CalculatorItemQueue<Double>!
    var operators: CalculatorItemQueue<Operator>!
    var sut: Formula!
    
    override func setUpWithError() throws {
        try super.setUpWithError()
        operands = .init()
        operators = .init()
        sut = .init(operands: operands, operators: operators)
    }

    override func tearDownWithError() throws {
        try super.tearDownWithError()
        operands = nil
        operators = nil
        sut = nil
    }
```

Reply:
Formula테스트에 대한 전반적인 수정을 하였고, 오류처리에 대한 케이스도 추가적으로 구현해줬습니다!